CRM/Utils/System/Base.php

```diff
@@ -37,13 +37,6 @@ abstract class CRM_Utils_System_Base {
    */
   public $is_wordpress = FALSE;
 
-  /**
-   * Does this CMS / UF support a CMS specific logging mechanism?
-   * @var bool
-   * @todo - we should think about offering up logging mechanisms in a way that is also extensible by extensions
-   */
-  public $supports_UF_Logging = FALSE;
```

Comment:
Have you checked `universe` for any uses of this? We sometimes get bitten by removing public properties that extensions or customizations were accessing, although this one is pretty obscure.

Reply:
I have not.


I hear people talk about this `universe` but I actually dont know how you check it... 

I couldnt think of a legit reason to access from outside core... though I suppose if you had an extension that provides logging on wordpress you might have iteracted with it?

Reply:
See https://docs.civicrm.org/dev/en/latest/tools/universe/#create-the-universe.

Reply:
Ok I just checked it and I don't see any references to it. So this looks good to go.
FYI `universe` is something you can build with `civibuild` and then just use your favorite flavor of grep to search it: https://docs.civicrm.org/dev/en/latest/tools/universe/